.github/workflows/qawolf/sync.js

```diff
@@ -116,6 +123,16 @@ async function writeFile(name, content) {
 }
 
 async function sync() {
+  // clear directory to remove old test files
+  await fs.rmdir("qawolf", { recursive: true });
+  await fs.mkdir("qawolf");
+
+  // copy getInbox helper
+  await fs.copyFile(
```

Comment:
@jperl - Why does this file live in the action if it's required by the tests? Is it different than `helpers.js`?

Reply:
I kept it as a separate file since it's pretty long. 

We wipe the qawolf folder now to remove old tests (for example the vscode test was renamed and updated something different) so that is why I copy it every sync.

Reply:
I'm okay with the copy but curious why one file is in the source tree and another is in the workflow.

Reply:
Source of truth is `.github/workflows/qawolf/*` it wipes out `qawolf/` every sync

Reply:
So `helpers.js` is sourced QA Wolf and added from the sync but `getInbox` isn't?